index.html

```diff
@@ -55,10 +57,21 @@ <h1>COLORANDOM</h1>
             <img src="assets/locked.png" class="boxes__locked hidden">
           </button>
         </box>
-        <p class="boxes__hex">#A4C4CA</p>
+        <box class="boxes__hex">#A4C4CA</box>
       </div>      
     </section>
+    <section class="buttons">
       <button class="buttons__new"> New Palette</button>
-    <script src="./main.js"></script>
+      <button class="buttons__save">Save Palette</button>
+    </section>
+  </section>
+<!-- saved section -->
+  <section class="saved">
+    <h2>Saved Palettes</h2>
+    <div class="saved__palettes">
+      <p class="saved__message">No saved palettes yet!</p>
+    </div>
+  </section>
 </body>
-</html>
\ No newline at end of file
+</html>
+<script src="./main.js"></script>
```

Comment:
Do we want to keep the script tag here or move it elsewhere?

Reply:
Oh, good call. I think it needs to be inside the closing body tag.

Reply:
Good idea, I've moved it up before the body and HTML closing tags and will push these changes.